packages/pluggableWidgets/tree-view-web/src/components/TreeView.tsx

```diff
@@ -78,7 +78,7 @@ export function TreeView({
             const treeViewIsExpanded = treeViewState === TreeViewState.EXPANDED;
             return showCustomIcon ? (
                 <Icon
-                    icon={treeViewIsExpanded ? collapseIcon : expandIcon}
+                    icon={treeViewIsExpanded ? expandedIcon : collapsedIcon}
```

Comment:
This is the most important change to address Diego's feedback, the rest is just to make the naming less confusing